### ISSUE TITLE
Update OpenStack Invoicing with new headers, updated SU Types and Rate/Cost

### ISSUE THREAD

**Issue by joachimweyl** [CCI-MOC/openstack-billing-from-db#7]
Please update the headers to match the header names we are using for the CSV.  The header list can be found in the [NERC August Invoice](https://docs.google.com/spreadsheets/d/1wE8BxMqIew300ex-YqoKcrec4NiRg6IPjH7A0KMx2Dw/edit#gid=536769863&range=1:1). 
The `SU type` and `Rate` are described [here](https://github.com/CCI-MOC/epics/issues/65#issuecomment-1688686685). 
Cost can be calculated by multiplying `Rate` by `SU Hours (GBhr or SUhr)`

**Comment by knikolla** [comment]
This only about renaming the headers rather than changing them, correct?

About the calculation, I have created a new issue #8

**Comment by joachimweyl** [comment]
correct, this is only about renaming in an effort to make combining all 3 easier down the line.

**Comment by knikolla** [comment]
Closed by #9 
Partial Issue #8 is resolved by #10
